### PR TITLE
added bulk insert spec

### PR DESCRIPTION
**Pull request by mikethebeer**

### CODE REVIEW

File: py/specs/insert_bulk.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    )
+"""
+
+drop_table_stmt = """
+    drop table t_bulk_insert
+"""
+
+def str_generator(char_length=100, size=10):
+    val_list = []
+    for i in range(0,size):
```
Comment (bogensberger) [inline py/specs/insert_bulk.py]: pep8: range(0, size): (missing whitespace after ,

File: py/specs/insert_bulk.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import random, string
+
+create_table_stmt = """
+    create table t_bulk_insert (
+      id integer,
```
Comment (bogensberger) [inline py/specs/insert_bulk.py]: is the id really required?

File: py/specs/insert_bulk.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    setup=Instructions(
+        statements=[create_table_stmt]
+    ),
+    teardown=Instructions(
+        statements=[drop_table_stmt]
+    ),
+    queries=[
+        {
+            'statement': 'insert into t_bulk_insert (id, value) values (?,?)',
+            'bulk_args': bulk_args_generator(),
```
Comment (bogensberger) [inline py/specs/insert_bulk.py]: the test will be more accurate if you generate the bulk_args upfront, now bulk_args generation is also part
of the benchmark

File: py/specs/insert_bulk.py [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+        value = ''.join(random.choice(string.ascii_uppercase + string.digits) for _ in range(char_length))
+        val_list.append(value)
+    return val_list
+
+def id_generator(size=10):
+    return [i for i in range(1, size + 1)]
+
+def bulk_args_generator(bulk_size=10000):
+    return list(zip(id_generator(bulk_size), str_generator(size=bulk_size)))
+
```
Comment (chaudum) [inline py/specs/insert_bulk.py]:
```py
def gen_args(x):
    return (x+1, random_string(), )

bulk_args = [gen_args(x) for x in range(10000)]
```

**Comment by chaudum** [comment]
add the spec to `tracks/latest.toml`

**Comment by chaudum** [comment]
otherwise LGTM

**Comment by mfussenegger** [comment]
Think it would be great to use a fixed seed to avoid having different data across benchmark runs?

**Pull request merged**